javascript/ql/src/semmle/javascript/dataflow/Configuration.qll

```diff
@@ -1038,6 +1038,25 @@ private predicate storeStep(
       receiverPropWrite(f, prop, mid)
     )
   )
+  or
+  // store in an immidiatly awaited function
```

Comment:
```suggestion
  // store in an immediately awaited function call
```

Reply:
Also this seems like an ideal candidate for caching in `FlowSteps`.

Reply:
> Also this seems like an ideal candidate for caching in FlowSteps.

It depends on `isAdditionalStoreStep`, so I don't think we can do that.